src/main/java/mate/academy/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -53,7 +53,7 @@ public Optional<MovieSession> get(Long id) {
     }
 
     @Override
-    public List<MovieSession> findAvailableSessions(Long movieId, LocalDate date) {
+    public List<MovieSession> findAvailableSessions(Long movieId, LocalDateTime date) {
```

Comment:
This fix is redundant, according the task description date type should be:
`findAvailableSessions(Long movieId, LocalDate date) {`
But MovieSession  `showTime` should have `LocalDateTime` type.
Please fix and check that solution works.